projects/projspec_template.md

```diff
@@ -0,0 +1,29 @@
+# Project: {CODENAME}
+
+[Long Project Name]
+
+## Campaign: {Campaign:CODENAME}
```

Comment:
Can a project belong to more than one campaign? If so, should this be a comma separated list?

Reply:
I hadn’t thought of that, but it seems like a good idea. Maybe bulleted list.

Reply:
I hadn’t thought of that, but it seems like a good idea. Maybe bulleted list.

Reply:
Resolved